upstairs/src/lib.rs

```diff
@@ -2602,8 +3023,18 @@ impl Upstairs {
      * If so, they lose the _  If not, then they will be removed.
      */
     fn ds_reconciliation(&self, _dst: &[Target], _lastcast: &mut u64) -> bool {
+        /*
+         * Reconciliation only happens during initialization.
+         */
+        let active = self.active.lock().unwrap();
+        if active.up_state == UpState::Active
```

Comment:
why not `active.up_state != UpState::Initializing`? if more states are added, then they could be missed being added to the if condition here

Reply:
Yeah, I think this may have been because I had more states for the upstairs, then changed my mind.
Done in 9608a17